src/main/java/domain/Cars.java

```diff
@@ -0,0 +1,52 @@
+package domain;
+
+import validation.ErrorMessage;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        validateDuplicateCars(cars);
+        this.cars = cars;
+    }
+
+    private void validateDuplicateCars(List<Car> cars) {
+        Set<Car> carSet = new HashSet<>();
```

Comment:
```suggestion
        Set<Car> carSet = new HashSet<>(cars);
```
항상 모두를 넣어준다면 생성시 초기화해주는 건 어떨까요?

Reply:
헉.. 놓쳤던 부분이네요 😅
수정하겠습니다 !